src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -925,6 +925,8 @@ a.vb
 
         <Fact>
         Public Sub LangVersion()
+            Dim myLangVersion = VisualBasicParseOptions.Default.LanguageVersion
```

Comment:
Move this local closer to its first use and rename to something more descriptive like "defaultLangVersion".